### ISSUE TITLE
Move vs-js material into ga-wdi-boston/ruby-vs-js-object-study

### ISSUE THREAD

**Issue by raq929** [ga-wdi-boston/ruby-object#10]
I started this repo, but it's still at the barely updated from study-template phase.

**Comment by jrhorn424** [comment]
> barely updated from study-template phase.

You mean the study?

**Comment by raq929** [comment]
Yeah, the one I linked to, IN MY MIND. Here it is: https://github.com/ga-wdi-boston/ruby-vs-js-object-study
